fares-data-build-tool/create-data
# feat: CFD-298 Fare day end global settings

# Description

-   Fare day end global settings

# Testing instructions

-   Check that you can create / edit the Fare Day End setting in Global settings
-   Check the additional GS pages show the Fare Day End info correctly

# Type of change
